internal/web_api.go

```diff
@@ -25,6 +26,12 @@ const apiVersion = "3.2"
 // It returns info whether to retry the request.
 type RetryStrategyFunc func(attempts uint, err error) bool
 
+// StatsSinkFunc if it is set then is invoked by each API request.
+//
+// It accepts one argumet: data - which is metrics data describing
+// requests action type, execution time of request and status of the request
```

Comment:
don't need to describe what `APICallStats` is here, it should be described where it's defined